graph/typeDefs.graphql

```diff
@@ -875,6 +877,9 @@ type CreateCommentResponse implements Response {
   # The comment that was created.
   comment: Comment
 
+  # Flags that was assigned during creation of the comment.
+  flags: [FlagAction]
```

Comment:
See above, should return `actions: [Action]` instead.

Reply:
👍